lib/transaction.py

```diff
@@ -736,6 +758,24 @@ def is_complete(self):
         s, r = self.signature_count()
         return r == s
 
+    @staticmethod
+    def _ecdsa_sign(sec, pre_hash):
```

Comment:
Would be nice if this had same call signature as _schnorr_sign.

In principle we can have `pubkey` be used to generate the `public_key` (MyVerifyingKey) object -- then the `public_key.verify_digest` is extra nice since it is actually checking that the signature comes from the expected pubkey (that keypairs had a correct mapping!).

Let me think about it, I can submit a PR later...

Reply:
Ah yeah sure I can see the advantage of the same call signature.  feel free to modify it then in your upcoming PR.. thanks man.